sdks/python/apache_beam/examples/complete/top_wikipedia_sessions.py

```diff
@@ -93,9 +93,8 @@ def expand(self, pcoll):
         | 'TopPerMonthWindow' >> beam.WindowInto(
             FixedWindows(size=THIRTY_DAYS_IN_SECONDS))
         | 'Top' >> combiners.core.CombineGlobally(
-            combiners.TopCombineFn(
-                10,
-                lambda first, second: first[1] < second[1])).without_defaults())
+            combiners.TopCombineFn(n=10,
+                                   key=lambda x: x[1])).without_defaults())
```

Comment:
was this a broken example on Py3?

Reply:
Could we pick a name for 'x' that describes the object in the context of this example?

Reply:
This change is needed by the TopCombineFn's [signature change](https://github.com/apache/beam/pull/14634/files#diff-c939f20e41fc848a26c5c8435fb348f118e81d9ced53dec490c1e5720a1be339R398) which removes `compare`.
And how about 'sessions_count' instead of 'x'? (updated at 3458e56 )

Reply:
sessions_count SGTM, thanks.
combiners.TopCombineFn is a public api, so this would mean an API change. If we had an API change, some users could be broken by this PR. But I suspect we don't have an API change for Py3 users, since we already disallowed compare on py3, and this example was already not working on Python 3, and will start working after this change. Does this sound correct?

Reply:
It seems not to sound correct.
It's true that combiners.Top.Of and PerKey disallow `compare` on Python 3 at #7842 , but combiners.TopCombineFn was left as is and still allow `compare` as an arg of the initializer.
Therefore this example have been working fine so far and Python 3 users could be affected by this PR ( I should have talked about this point in advance )

Reply:
Ok, thanks. I think it's ok to proceed with this change, but I would mention the breaking change in CHANGES.md.

Reply:
Ah, I see. It's necessary.
And is it OK to edit CHANGES.md in this PR?
Rebase and force-push are needed if doing so since there is an update in CHANGES.md.

Reply:
rebase and force-push is fine. Feel free to squash all commits since they have been reviewed and you are not adding new code.

Reply:
Thanks, update CHANGES.md at 565248d